src/app/zap-templates/zcl/zcl.json

```diff
@@ -285,8 +286,7 @@
             "ActiveModeThreshold",
             "RegisteredClients",
             "ICDCounter",
-            "ClientsSupportedPerFabric",
-            "MaximumCheckInBackOff"
+            "ClientsSupportedPerFabric"
```

Comment:
This change is wrong.  MaximumCheckInBackOff is in fact implemented in the relevant cluster, and is implemented via AttributeAccessInterface.

If the tool made this change, the tool needs to be fixed.

Reply:
I'm not sure what's going on, if it's user error on my part where I have some stale file messing things up (Hasty was unable to repro), or if it's an issue with Alchemy

I've manually updated the files properly & will do some Alchemy debugging to see what's up